ethereum/contracts/relayer/coreRelayer/CoreRelayerMessages.sol

```diff
@@ -363,13 +325,14 @@ abstract contract CoreRelayerMessages is CoreRelayerGetters {
      * @param provider relay provider
      * @return targetAmount amount of target chain currency
      */
-    function convertReceiverValueAmount(uint256 sourceAmount, uint16 targetChain, IRelayProvider provider)
+    function convertReceiverValueAmountToTarget(uint256 sourceAmount, uint16 targetChain, IRelayProvider provider)
         internal
         view
         returns (uint256 targetAmount)
     {
         (uint16 buffer, uint16 denominator) = provider.getAssetConversionBuffer(targetChain);
 
+        // todo: akshaj why is `uint256(0) + ...` present?
```

Comment:
it is present so that we don't hit the overflow limit of the uint16 when adding 'denominator' with 'buffer'